tutorials/site-monitoring-foundations.ipynb

```diff
@@ -0,0 +1,2737 @@
+{
```

Comment:
<p>you can delete the first line, I think</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/PlanetaryComputerExamples/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>